src/main/kotlin/no/nav/familie/ef/mottak/service/OppgaveService.kt

```diff
@@ -215,6 +227,13 @@ class OppgaveService(private val integrasjonerClient: IntegrasjonerClient,
         }
     }
 
+    private fun utledBehandlesAvApplikasjonEttersending(ettersending: Ettersending): BehandlesAvApplikasjon {
+        val søknadId = EttersendingMapper.toDto<EttersendingDto>(ettersending).ettersendingForSøknad?.søknadId
+        val søknad = søknadService.get(søknadId ?: throw Error("Fant ikke søknad for ettersending"))
```

Comment:
Er det mulig for ettersendinger uten søknadId å komme hit?

Reply:
Dette tryner ja, men taskworkeren klager ikke 😱 Står bare som "plukket"
https://familie-prosessering.dev.nav.no/service/familie-ef-mottak/gruppert?statusFilter=ALLE&callId=2002e3599a66489cb1f4b34b9e3c29d5

Reply:
Fikset buggen nå, men burde kanskje sjekke ut prosessering - denne burde vel blitt satt til feilet?

Reply:
Ja, tror det :suspect: